### ISSUE TITLE
Fix licenseUrl element in nuspec, will be deprecated

### ISSUE THREAD

**Issue by mikkelbu** [nunit/nunit#3186]
Warning when generating nuget packages. See e.g. https://ci.appveyor.com/project/CharliePoole/nunit/builds/22954283
```
========================================
PackageFramework
========================================
Attempting to build package from 'nunit.temp.nuspec'.
Successfully created package 'C:\projects\nunit\package\NUnit.3.12.0-ci-06156-pr-3184.nupkg'.
WARNING: NU5125: The 'licenseUrl' element will be deprecated. Consider using the 'license' element instead.
Attempting to build package from 'nunitlite.temp.nuspec'.
Successfully created package 'C:\projects\nunit\package\NUnitLite.3.12.0-ci-06156-pr-3184.nupkg'.
WARNING: NU5125: The 'licenseUrl' element will be deprecated. Consider using the 'license' element instead.
```

**Comment by parrainc** [comment]
If no one have done something yet, I can take some few mins to fix this later today.

@mikkelbu question, according to the docs [here](https://docs.microsoft.com/en-us/nuget/reference/nuspec#license) it says that we should use `<license type="expression">..</license>` with a SPDX license id (in nunit's case would be MIT). So, just wondering if it would be ok changing the LICENSE.txt file path with this identifier in the nuspec file? Otherwise I'll have to add `type="file"`

I'll be waiting for response to create the PR.

**Comment by mikkelbu** [comment]
We would love a PR @parrainc. Regarding of whether we should go with `type="expression"` or `type="file"` I'm leaning towards `file`, but that is mostly because I'm no expert in licensens. @nunit/framework-team do any of you have any opinion about this? (will linking to a general MIT licens from nuget be fine or should we rather link to our file that mentions Charlie and Rob?)

**Comment by parrainc** [comment]
@mikkelbu me as well, I'm leaning for the second one as well since in the .nuspec files the LICENSE.txt is being deployed as part of the nuget package, and nuget recommends that if we're going to use type file for licenses. 

PR is created with type="file", if the team has any suggestion/opinion on going with file or expression, I can update the same.

**Comment by mikkelbu** [comment]
This was fixed in #3192.
